buildSrc/src/main/kotlin/buildsrc/conventions/publishing.gradle.kts

```diff
@@ -13,14 +13,13 @@ import org.gradle.api.plugins.JavaBasePlugin.DOCUMENTATION_GROUP
 
 plugins {
     signing
-    `maven-publish`
+    id("com.vanniktech.maven.publish")
 }
 
 
 //region Publication Properties
 // can be set in gradle.properties or environment variables, e.g. ORG_GRADLE_PROJECT_snake-kmp.ossrhUsername
-val ossrhUsername = providers.gradleProperty("snake-kmp.ossrhUsername")
-val ossrhPassword = providers.gradleProperty("snake-kmp.ossrhPassword")
+// TODO remove the below?
```

Comment:
The `com.vanniktech.maven.publish` does not require the `sonatypeReleaseUrl` or `isReleaseVersion` - we can safely remove them.

Also, this block is not needed as the plugin configures everything
```
afterEvaluate {
    // Register signatures in afterEvaluate, otherwise the signing plugin creates
    // the signing tasks too early, before all the publications are added.
    signing {
        sign(publishing.publications)
    }
}
```

as well as this region: `Fix Gradle warning about signing tasks using publishing task outputs without explicit dependencies` - plugin solves that too

Reply:
Thanks for the hints! Removed.